gwdatafind/http.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
-# Copyright (C) 2012-2015  Scott Koranda, 2015+ Duncan Macleod
+# Copyright (C) 2012-2015  Scott Koranda, 2015+ Duncan Macleod,
+# 2021 Duncan Brown
```

Comment:
My copyright entry is definitely wrong and should refer to LSU and Cardiff University and not me as an individual. @duncan-brown do you actually get to claim personal copyright ownership over your work IP while at Syracuse?